config/packages/rate_limiter.yaml

```diff
@@ -0,0 +1,7 @@
+framework:
+    rate_limiter:
+        forgot_password_email:
+            policy: sliding_window
+            limit: 1
+            interval: 5 minutes
+            lock_factory: null
```

Comment:
did you make a PR so this isn't required in 7.3?

Reply:
Yes, I can mark for removal